monitoring/airgap/airgap-tempo-values.yaml

```diff
@@ -0,0 +1,4 @@
+image:
+  repository: __AIRGAP_REGISTRY__/tempo/tempo
+  pullSecrets:
+  - name: __AIRGAP_IMAGE_PULL_SECRET_NAME__
```

Comment:
See above; the correct key may be **tempo.pullSecrets** (although I also see a **serviceAccount.imagePullSecret** key).  Please confirm we're setting the correct keys.